src/graphql/schema/post.js

```diff
@@ -0,0 +1,82 @@
+const sharedPostInputFields = `
+  latest: String!
+  page: Int!
+  pageSize: Int!
+`;
+
+export default `
+  type Post {
```

Comment:
Please use camelCase convention rather then underscore case. I know database schema uses underscore case but this is just because it's a MySQL convention